draft-ietf-rats-corim.md

```diff
@@ -1980,7 +1941,7 @@ If a field is not present in the stateful environment `environment-map` then the
 
```

Comment:
I think the paragraphs above need to be enhanced to say how measurement-values-maps are selected for comparison. Example text for this might be:

```suggestion
The Verifier initializes a temporary "candidate entries" variable with all entries in the ACS where the stateful environment `environment-map` and measured element identifier is a subset of the ACS `environment-map` and measured element identifier.

A stateful environment `environment-map` is a subset of an ACS entry `environment-map` if each field (for example `class`, `instance` etc.) which is present in the stateful environment `environment-map` is also present in the ACS entry, and the CBOR encoded field values in the stateful environment and ACS entry are binary identical.
If a field is not present in the stateful environment `environment-map` then the presence of, and value of, the corresponding ACS entry field does not affect whether the `environment-map`s are subsets.
A stateful environment measured element identifier is a subset of an ACS entry measured element identifier if their CBOR encodings are binary identical.
```